### PR TITLE
Made modifications to match today's SimpleReport

### PR DESCRIPTION
**Pull request by jimduff-usds**
This PR does ...

- add Patient_email
- add placeholder for facility email (empty string) Ordering_facility_email
- remove Patient_lookup_ID
- In SimpleReport incoming, rename Testing_lab_specimen_ID to Result_ID (this stores the unique row id)
- In outgoing data, this data keeps the same name it had before (eg Specimen_ID in AZ data)

This is to match with: https://github.com/CDCgov/prime-simplereport/pull/330

## Checklist
- [X] Tested locally?
- [ ] Added new dependencies?
- [ ] Updated the docs?
- [X] Added a test?  (Still in a script:  ./quick-test.sh)
- [ ] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## To Be Done

- Move script test to .kt test.

### CODE REVIEW

File: prime-router/metadata/schemas/AZ/az-covid-19-csv.schema [hunk 1]
```diff
@@ -14,6 +14,7 @@ elements:
 
   # Nonstandard
   - name: organization_and_lab
+    type: TEXT
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/AZ/az-covid-19-csv.schema:17]: Tests detected this issue: the faker expects TYPES for nonstandard fields.

File: prime-router/metadata/schemas/AZ/az-covid-19-csv.schema [hunk 2]
```diff
@@ -147,6 +149,7 @@ elements:
     - name: Provider_Phone_Number
 
   - name: testing_lab_specimen_id
+    mapper: use(placer_order_id)
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/AZ/az-covid-19-csv.schema:152]: Hopefully someday we'll get a real specimen_id from SR.

File: prime-router/metadata/schemas/AZ/pima-az-covid-19-csv.schema [hunk 3]
```diff
@@ -21,11 +21,6 @@ elements:
   - name: patient_id
     csvFields: [{name: Patient_ID}]
 
-  # In SimpleReport, lookup_id is the ID the user entered into the app.
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/AZ/pima-az-covid-19-csv.schema:24]: SR wasn't really using this, so they took it out.

File: prime-router/quick-test.sh [hunk 4]
```diff
@@ -1,48 +1,127 @@
 #!/usr/bin/env bash
 
-# Run prime on csv file and compare outputs to actuals
+# Learn from my mistakes:
```
Comment (jimduff-usds) [inline prime-router/quick-test.sh:3]: Yes, editing data by hand is a pain, but it found a lot of bugs.

File: prime-router/quick-test.sh [hunk 5]
```diff
@@ -1,48 +1,127 @@
...
+parse_prime_output_for_filename "$text" "/az"
+actual_az4=$filename
+compare_files "AZ->AZ" $actual_az3 $actual_az4
+
+# Pima again again.  Compare the two actuals
+echo Test sending Pima data generated from fake data into its own Schema:
+text=$(./prime --input_schema az/pima-az-covid-19 --input $actual_pima3 --output_dir $outputdir)
+parse_prime_output_for_filename "$text" "/pima"
+actual_pima4=$filename
+compare_files "PIMA->PIMA" $actual_pima3 $actual_pima4
```
Comment (jimduff-usds) [inline prime-router/quick-test.sh:124]: I hope y'all try running my little test script.

**Review by MauriceReeves-usds — APPROVED** [review]

**Pull request merged**